docs/source/asr/speaker_diarization/datasets.rst

```diff
@@ -1,72 +1,133 @@
 Datasets
 ========
-Check out page :doc:`Speech Classification Datasets <../speech_classification/datasets>` and :doc:`Speaker Recogniton Datasets <../speaker_recognition/datasets>` 
-for preparing datasets for training and validating VAD and speaker embedding models respectively.
 
-For Speaker Diarization inference, ``diarizer`` expects following arguments :
+This page is about preparing input dataset for diarization inference. To train or fine-tune speaker diarization system, the speaker embedding extractor should be trained or fine-tuned separately. Check out page :doc:`Speech Classification Datasets <../speech_classification/datasets>` and :doc:`Speaker Recogniton Datasets <../speaker_recognition/datasets>` 
```

Comment:
To train or fine-tune speaker diarization system, the speaker embedding extractor should be trained or fine-tuned separately. 

How about add VAD to this sentence as well

Reply:
While your suggested sentence is true, adding VAD here loses the point of this sentence. This sentence is explaining users who are finding a way to adapt diarization model on their own data.